addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -1185,7 +1185,8 @@ public Action Command_ToBonus(int client, int args)
 		zoneGrp = 1;
 
 	g_bInBonus[client] = true;
-	g_iInBonus[client] = zoneGrp;
+	if (g_mapZoneGroupCount > zoneGrp)
```

Comment:
This stops a bug where if you type `/b <anyNumber>` that number will be assigned to `g_iInBonus[client]`

Have not noticed it breaking something on live version, but now this variable does not get appended with a new number if there are not that many bonus zones.

Reply:
![image](https://user-images.githubusercontent.com/74899888/169240100-67e8ef0f-848b-4ce6-8e5a-110484fc5b3c.png)



**N I C E**